docs/getting-started-guides/scratch.md

```diff
@@ -66,21 +67,25 @@ the node is added.  A process in one pod should be able to communicate with
 another pod using the IP of the second pod.  This connectivity can be
 accomplished in two ways:
 
-- Configure network to route Pod IPs
+- **Configure underlay network to route Pod IPs**
   - Harder to setup from scratch.
   - Google Compute Engine ([GCE](/docs/getting-started-guides/gce)) and [AWS](/docs/getting-started-guides/aws) guides use this approach.
-  - Need to make the Pod IPs routable by programming routers, switches, etc.
-  - Can be configured external to Kubernetes, or can implement in the "Routes" interface of a Cloud Provider module.
+  - Need to make the Pod IPs routable by programming routers, switches, etc.  
+    - This can be done in a few different ways:
+      - Implement in the "Routes" interface of a Cloud Provider module.
```

Comment:
(Requires building your own fork of Kubernetes or contributing code to the project, right?)